doc/source/database/cratedb.rst

```diff
@@ -0,0 +1,518 @@
+distributed across the nodes of a cluster. Each shard in CrateDB is a Lucene index
+broken into segments and stored on the filesystem.
+
+The clustering subsystem, effectively distributing data amongst multiple storage nodes,
+has been implemented and maintained in CrateDB from the very beginning, based on prior
+art technology implementations from Elasticsearch.
+
+This procedure is effective for storing and retrieving large amounts of data, in the
+range of billions of records, and terabytes of data.
+
+On data retrieval, CrateDB's blazing-fast distributed query execution engine
+parallelizes query workloads across the whole cluster.
+
+Complex queries
+===============
+
+By using the SQL query language, CrateDB provides an advanced query execution layer,
+unlocking complex querying capabilities like date range filtering, sub-selects,
+aggregations, JOINs, UNIONs, and CTEs, all within the same SQL statement.
+
+Query API
+=========
+
+- CrateDB provides an `HTTP endpoint`_ that can be used to submit SQL queries.
+  As such, any HTTP client, like curl or Postman, can be used to communicate with CrateDB.
+
+- The standards-based `PostgreSQL wire protocol`_ interface unlocks compatibility
+  with a wide range of client applications which can talk to PostgreSQL servers.
+
+Ecosystem
+=========
+
+CrateDB offers a wide range of connectivity options. In general, any PostgreSQL-
+compatible driver or framework can be used to connect to CrateDB.
+
+- `Overview of CrateDB drivers and adapters`_
+- `Overview of CrateDB integration tutorials`_
+
+User interface
+==============
+
+CrateDB offers both a graphical-, and a commandline-based user interface. In general,
+any PostgreSQL-compatible applications and systems can be used to connect to CrateDB.
+
+- CrateDB Admin, a graphical, web-based user interface, is built into CrateDB.
+- ``crash`` is a command-line based terminal program, similar to ``psql``, but
+  with a bit more convenience.
+
+
+.. _cratedb-query-examples:
+
+**************
+Query examples
+**************
+
+This section demonstrates a few query examples from CrateDB's documentation.
+
+Typical queries
+===============
+
+.. code-block:: sql
+
+    -- Aggregations with date range filtering and
+    -- time bucketing using specified intervals.
+    SELECT
+        DATE_BIN('5 min'::INTERVAL, time::TIMESTAMPTZ, 0) AS time,
+        MEAN(fields['humidity']) AS humidity
+    FROM
+        readings
+    WHERE
+        time > NOW() - '1 hour'::INTERVAL
+    GROUP BY
+        time
+    ORDER BY
+        time;
+
+.. code-block:: sql
+
+    -- An SQL DDL statement defining a custom schema for holding sensor data.
+    CREATE TABLE iot_data (
+      timestamp TIMESTAMP WITH TIME ZONE,
+      sensor_data OBJECT (DYNAMIC) AS (
+        temperature FLOAT,
+        humidity FLOAT,
+        location OBJECT (DYNAMIC) AS (
+          latitude DOUBLE PRECISION, longitude DOUBLE PRECISION
+        )
+      )
+    );
```

Comment:
That's the only SQL DDL statement within the "query examples" section. Others, including a few other CrateDB special data types, may be sensible.

Reply:
Any chance of using `GEO_POINT` maybe?